HDPDB/products/models.py

```diff
@@ -0,0 +1,93 @@
+
+from django.db import models
+
+from users.models import User
+
+
+class Main_Category(models.Model):
```

Comment:
class 명에는 언더바(_)를 사용하지 않습니다~
```suggestion
class MainCategory(models.Model):
```